File: 1470-shuffle-the-array/1470-shuffle-the-array.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function(nums, n) {
  const a1 = nums.slice(0, n)
  const a2 = nums.slice(n)
  const result = []
  for (let i = 0; i < n; i++) {
    result.push(a1[i])
    result.push(a2[i])
  }
  return result
};

File: 1470-shuffle-the-array/1470-shuffle-the-array.go
func shuffle(nums []int, n int) []int {
  ans := make([]int, n*2)
  for i := 0; i < n; i ++ {
    ans[2*i] = nums[i]
    ans[2*i+1] = nums[n+i]
  }  
  return ans
}